File: models/OrderDeliveryType.js
const utils = require('../utils/utils');

module.exports = {
        fields: (key) => (
            {
                label: `Способ доставки заказа:  * `DELIVERY` — курьерская доставка.  * `PICKUP` — самовывоз.  * `POST` — почта.  * `DIGITAL` — для цифровых товаров.  * `UNKNOWN` — неизвестный тип.  - [${key.replaceAll('__', '.')}]`,
                choices: [
                    'DELIVERY',
                    'PICKUP',
                    'POST',
                    'DIGITAL',
                    'UNKNOWN',
                ],
            }
        )
    }

File: ympa_fsharp_functions_server/src/model/OrderDeliveryType.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json

module OrderDeliveryType =

  //#region OrderDeliveryType

  let DELIVERYEnum = "DELIVERY"
  let PICKUPEnum = "PICKUP"
  let POSTEnum = "POST"
  let DIGITALEnum = "DIGITAL"
  let UNKNOWNEnum = "UNKNOWN"
  type OrderDeliveryType = string

  //#endregion
